docs/serving/model-serving-get-started.ipynb

```diff
@@ -850,6 +850,145 @@
     "- Steps B and C are filter steps that complement each other. For example B passes events where key < X, and C passes events where key >= X. The resulting DF contains the exact event ingested, since each event was handled once on one of the branches.\n",
```

Comment:
<p>use <code>project.set_function</code> instead of  <code>mlrun.code_to_function</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
@alonmr Where? And why?

Reply:
In general we want users to use `project.set_function` and we want to keep `mlrun.code_to_function` mainly for internal use because the former sets the function for the project among other things. This is where you create the serving function